src/ShortestPaths.h

```diff
@@ -0,0 +1,776 @@
+/// Causal Dynamical Triangulations in C++ using CGAL
```

Comment:
I stopped working on ShortestPaths.h after I realized we needed to complete the S3ErgodicMoves.h first. I will resume soon. But as of now, there are no test cases.